AppLovinAdapter/src/main/java/com/chartboost/mediation/applovinadapter/AppLovinAdapter.kt

```diff
@@ -633,23 +633,31 @@ class AppLovinAdapter : PartnerAdapter {
         return suspendCoroutine { continuation ->
             val rewardedAd = AppLovinIncentivizedInterstitial.create(appLovinSdk)
 
+            var isUserVerified: Boolean? = null
```

Comment:
if neither is called, we okay with the reward still going through?

Reply:
I think so. Let me find the AppLovin FAQ that I used to decide this

Reply:
Okay, so there are two places that slightly conflict.


https://github.com/AppLovin/AppLovin-MAX-SDK-Android/blob/306ce96844d0558e4287a8fc2df3f5fa059ecfcb/AppLovin%20MAX%20Demo%20App%20-%20Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/rewarded/RewardedVideosActivity.kt#LL100C1-L101C128:
```
        // AppLovin servers validated the reward. Refresh user balance from your server.  We will also pass the number of coins
        // awarded and the name of the currency.  However, ideally, you should verify this with your server before granting it.
```

https://support.applovin.com/hc/en-us/articles/4403802571661-When-is-the-best-time-to-reward-the-user-:
```
One possibility is rewarding the user if the reward was verified for this video play, but only upon completion of the video. For this method you can set a boolean flag to indicate that the reward was verified, and check its value upon video completion.
```

Not a huge difference between them, but to me the first sounds a bit more permissive of the default being to reward them.